domain/domain.go

```diff
@@ -711,6 +717,15 @@ func (do *Domain) NotifyUpdatePrivilege(ctx context.Context) {
 	}
 }
 
+func recoverInDomain(funcName string) {
```

Comment:
why we need the funcName parameter? GetStack can't distinguish which goroutine throw the panic?

Reply:
It's easier to see in the first line than in the stack trace.